### DISCUSSION TITLE
How to instantly switch between VST and VR modes

### DISCUSSION THREAD

**Discussion by jike2018** [StereoKit/StereoKit#930] [Q&A]
When I want to use VST mode, I will set bledPreference=DisplayBlended.Blend in SKSettings during initialization. However, how should I switch between VR and VST when the app is running?

**Comment by maluoi** [comment]
You can use [`Device.DisplayBlend`](https://stereokit.net/Pages/StereoKit/Device/DisplayBlend.html) at runtime :)
